Common/Securities/SecurityTransactionManager.cs

```diff
@@ -253,33 +263,26 @@ public bool WaitForOrder(int orderId)
             return true;
         }
 
-        /// <summary>
-        /// Get a list of all open orders.
-        /// </summary>
-        /// <returns>List of open orders.</returns>
-        public List<Order> GetOpenOrders()
```

Comment:
Was this method completely removed?

Reply:
With the new `public List<Order> GetOpenOrders(Func<Order, bool> filter = null)` both previous API methods `public List<Order> GetOpenOrders()` and `public List<Order> GetOpenOrders(Func<Order, bool> filter)` are covered 👍